src/storage/query/QueryEdgePropsProcessor.cpp

```diff
@@ -26,9 +26,15 @@ kvstore::ResultCode QueryEdgePropsProcessor::collectEdgesProps(
     if (ret != kvstore::ResultCode::SUCCEEDED) {
         return ret;
     }
+
+    auto schema = edgeSchema_.find(edgeKey.edge_type);
```

Comment:
For QueryEdgeProcessor,  it only support one edgeType one request.  So you could pass the related edge schema directly into collectEdgesProps

Reply:
Got it.